apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -39,6 +42,14 @@ export class TriggerEvent {
       },
     });
 
+    this.logger.assign({
+      transactionId: command.transactionId,
+      environmentId: command.environmentId,
+      organizationId: command.organizationId,
+    });
+
+    await this.validateTransactionIdProperty(command.transactionId, command.organizationId, command.environmentId);
```

Comment:
Why is this line here was added? It doesn't look like it exists on next branch

Reply:
@Cliftonz saw that you resolved this issue, but seems like this line is still here, could you elaborate?